hack/ci-common.sh

```diff
@@ -19,7 +19,13 @@ set -o errexit
 export_logs() {
```

Comment:
Can we rename this function now that it is no longer exporting logs only? `export_artifacts` maybe?

Reply:
Fixed